app/User.php

```diff
@@ -1,23 +1,24 @@
 <?php
 
+
 namespace App;
 
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Http\Request;
+
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\Notifiable;
 
-class User extends Authenticatable
+class User extends Model //Authenticatable,
 {
+
+
     use Notifiable;
 
-    /**
-     * The attributes that are mass assignable.
-     *
-     * @var array
-     */
-    protected $fillable = [
-        'name', 'email', 'password',
-    ];
+    //protected $table = 'user';
+    //id, name, role, lastName, age, email, password
+    protected $fillable = ['name','lastName','age','email','password'];
```

Comment:
Remember that the properties of the models will be named in singular, lowercase and with snake_case.
Change lastName to last_name please.